tests/baselines/reference/ambientShorthand_reExport.js

```diff
@@ -20,11 +20,19 @@ x($);
 "use strict";
 exports.__esModule = true;
 var jquery_1 = require("jquery");
-exports.x = jquery_1.x;
+Object.defineProperty(exports, "x", { enumerable: true, get: () => jquery_1.x });

```

Comment:
I believe this test and every other test that doesn't have a `// @target` defaults to ES3, so many of these test results are actually incorrect as they would not work in the target environment (where `Object.defineProperty` is not defined).